java gr 8/homework/05homework5_java_oop_implementation/src/main/java/siit/java8/evppm2/Dealership.java

```diff
@@ -0,0 +1,20 @@
+package siit.java8.evppm2;
+
+/**
+ * Created by Narci on 8/26/2017
+ */
+public class Dealership {
+
+    static EvCar[] evCars = new EvCar[ 9 ];
+
+    public Dealership ( ) {
+    }
+
+    public static void initEvCars ( int position, int price, String manufacturer, String model ) {
```

Comment:
This should be a non-static method and should be called addCar.

Reply:
public class Dealership {

   static EvCar[] evCars = new EvCar[ 9 ];
    public  int[] addEvCars;

    public Dealership () {

    }

    public static void addEvCars ( int position, int price, String manufacturer, String model ) {
        evCars[ position ] = new EvCar ();
        evCars[ position ].price = price;
        evCars[ position ].manufacturer = manufacturer;
        evCars[ position ].model = model;
    }
}